.github/task-definition.json

```diff
@@ -48,7 +48,7 @@
         },
         {
           "name": "ERC4337_BUNDLER_SERVICE_NAME",
-          "value": "VAR_ERC4337_BUNDLER_SERVICE_NAME"
+          "value": "Bundler"
```

Comment:
can we cross check, it should be have either prefix or postfix as per env. e.g.; `Bundler-Devnet`, `Bundler-Testnet`

Reply:
I don't think it has to have any specific prefix. We already have the logs properly separated in buckets and they all include `environment` label in all the logs. We can filter that way

`environment = dev` 
`environment = test`